README.md

```diff
@@ -1,3 +1,11 @@
+# Local Deployment
+1. Create a .env file with github account details:
+    - CLIENT_ID =
```

Comment:
I find a good way of making this easy is to create a .sample file (or any renamed file).

The instructions can then be just to take a copy of that file and fill in the keys